workflows/prognostic_c48_run/emulation/emulation/emulate.py

```diff
@@ -0,0 +1,105 @@
+import sys
+
+# Tensorflow looks at sys args which are not initialized
+# when this module is loaded under callpyfort, so ensure
+# it's available here
+if not hasattr(sys, "argv"):
+    sys.argv = [""]
+
+import f90nml  # noqa: E402
+import logging  # noqa: E402
+import os  # noqa: E402
+import tensorflow as tf  # noqa: E402
+
+from .debug import print_errors  # noqa: E402
+from ._filesystem import get_dir  # noqa: E402
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.DEBUG)
+
+TF_MODEL_PATH = None  # local or remote path to tensorflow model
+NML_PATH = None
+DT_SEC = None
+ORIG_OUTPUTS = None
+MODEL = None
+
+
+@print_errors
+def _load_environment_vars_into_global():
+
+    global TF_MODEL_PATH
+    global NML_PATH
+
+    cwd = os.getcwd()
+    TF_MODEL_PATH = os.environ.get("TF_MODEL_PATH", None)
+    NML_PATH = os.path.join(cwd, "input.nml")
+
+
+@print_errors
+def _load_nml():
+    namelist = f90nml.read(NML_PATH)
+    logger.info(f"Loaded namelist for emulation from {NML_PATH}")
+
+    return namelist
+
+
+@print_errors
+def _get_timestep(namelist):
+    return int(namelist["coupler_nml"]["dt_atmos"])
+
+
+@print_errors
+def _load_tf_model() -> tf.keras.Model:
+    logger.debug(f"Loading keras model: {TF_MODEL_PATH}")
```

Comment:
Suggest making this `info` level since it is pretty significant information (and is currently the only way to tell what model is used since the env variables are not stored in the outputs).

```suggestion
    logger.info(f"Loading keras model: {TF_MODEL_PATH}")
```